webviz_subsurface/_datainput/well_completions.py

```diff
@@ -61,3 +70,41 @@ def read_well_attributes(
             for well_data in file_content["wells"]
         }
     return None
+
+
+def read_stratigraphy(
+    ensemble_path: str, stratigraphy_file: str
+) -> Optional[List[Dict]]:
+    """Searches for a stratigraphy json file on the scratch disk. \
+    If a file is found the content is returned as a list of dicts.
+    """
+    for filename in glob.glob(f"{ensemble_path}/{stratigraphy_file}"):
+        return json.loads(Path(filename).read_text())
+    return None
+
+
+def get_ecl_unit_system(ensemble_path: str) -> Optional[str]:
+    """Returns the unit system of an eclipse deck. The options are \
+    METRIC, FIELD, LAB and PVT-M.
+
+    If none of these are found, the function returns None, even though \
+    the default unit system is METRIC. This is because the unit system \
+    keyword could be in an include file.
+    """
+
+    def ecldata_has_unit_system(unit_system: str, lines: list) -> bool:
+        for line in lines:
+            if line.startswith(unit_system):
+                return True
+        return False
```

Comment:
```suggestion
        return any(line.startswith(unit_system) for line in lines)
```
Maybe the function is not needed if can be this short and is used only once below?

Reply:
smart!